app/build.gradle

```diff
@@ -36,7 +36,11 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    //location service library
```

Comment:
This comment is not needed.  The library description is a given, from the name of the library itself.  Remove comment to reduce clutter.